frontend/coprs_frontend/coprs/views/apiv3_ns/__init__.py

```diff
@@ -23,6 +24,29 @@
 apiv3_ns = flask.Blueprint("apiv3_ns", __name__, url_prefix="/api_3")
 
 
+# Somewhere between flask-restx 1.0.3 and 1.1.0 this change was introduced:
+# > Initializing the Api object always registers the root endpoint / even if
+# > the Swagger UI path is changed. If you wish to use the root endpoint /
+# > for other purposes, you must register it before initializing the Api object.
+# TODO That needs to get fixed and then we can move this route to apiv3_general
```

Comment:
There's never a good time to do this.  Can we have an issue for this (we should create a new api end point at least, and twist our library to use it).

Reply:
I linked the issue in the code comment